connector-definition/scripts/check-reqs.ps1

```diff
@@ -0,0 +1,27 @@
+$ErrorActionPreference = "Stop"
+
+$minimumNodeVersion = 20
+
+if (-not (Get-Command "node" -ErrorAction SilentlyContinue)) {
+  Write-Host "node could not be found. Please install NodeJS v$minimumNodeVersion+."
+  exit 1
+}
+
+$nodeVersion = & node --version
+if ($nodeVersion -match "^v(\d+)\.") {
+  $majorVersion = $Matches[1]
+  if ($majorVersion -lt $minimumNodeVersion) {
+    Write-Host "Detected Node.js version $nodeVersion on the PATH. The minimum required version is v$minimumNodeVersion."
+    exit 1
+  }
+}
```

Comment:
Should this be specified as a constant somewhere so that is can be shared between scripts, etc?

Reply:
`$minimumNodeVersion` I mean

Reply:
Yeah potentially, but unfortunately the node version (or something linked to the node version) is used in a lot of disparate places across the codebase that can't be unified into a single configurable value (eg. package.json, node ts types version, .nvm, Dockerfiles, tsconfig templates, tsconfig defaults inside the code, etc).

I could unify it in the scripts, but it would mean having to read the value from a file, which seems like more script complexity to simply reduce two tiny definitions into one. It doesn't feel worth it, given the load of other places that cannot not use this value and still have to be corrected by hand. If you feel strongly that it _is_ worth it for these two places, I can change it very easily.